src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,61 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { getMovie } from '../../api';
+import { ResponseError } from '../../types/ReponseError';
+import { convert } from '../Convertation';
+import { MovieCard } from '../MovieCard/MovieCard';
+
+interface Props {
+  addMovie: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movieLoading, setMovieLoading] = useState(false);
```

Comment:
```suggestion
  const [isMovieLoading, setIsMovieLoading] = useState(false);
```
![image](https://github.com/mate-academy/react_movies-list-fetch-movies/assets/96409563/fb2ec556-28d4-4292-a79b-5014225f652e)

Reply:
Done, thank you for your review!